packages/frontend/src/pages/ParkingSpaces/components/create-applicant-for-listing/CreateApplicantForListing.tsx

```diff
@@ -152,11 +152,22 @@ export const CreateApplicantForListing = (props: Props) => {
                       <ContactInfo tenant={tenantQuery.data?.tenant ?? null} />
                     )}
                     {tenantQuery.isLoading && <ContactInfoLoading />}
-                    {tenantQuery.error && (
-                      <Typography color="error">
-                        Något gick fel. Kontakta support.
-                      </Typography>
-                    )}
+                    {tenantQuery.error &&
+                      tenantQuery.error?.response?.data?.type ===
+                        'no-valid-housing-contract' && (
```

Comment:
I feel like these conditionals look too busy. Can we assign to a var instead? I.e
```suggestion
                    {hasNoValidHousingContract && (
```